client/v2/algod/tealDisassemble.go

```diff
@@ -0,0 +1,23 @@
+package algod
+
+import (
+	"context"
+
+	"github.com/algorand/go-algorand-sdk/client/v2/common"
+	"github.com/algorand/go-algorand-sdk/client/v2/common/models"
+)
+
+// TealDisassemble given the base64 encoded program bytes, return the TEAL source
+// code in plain text. This endpoint is only enabled when a node's configuration
+// file sets EnableDeveloperAPI to true.
+type TealDisassemble struct {
+	c *Client
+
+	source string
+}
+
+// Do performs the HTTP request
+func (s *TealDisassemble) Do(ctx context.Context, headers ...*common.Header) (response models.DisassembleResponse, err error) {
+	err = s.c.get(ctx, &response, "/v2/teal/disassemble", nil, headers)
```

Comment:
According to the link jason shared, shouldn't this be a POST and include the bytes body?
https://github.com/algorand/go-algorand/blob/aa82b9a6e78dd38c8115120ca90b817f6be73ef3/daemon/algod/api/algod.oas2.json#L1441

Reply:
Sorry, this is a stale change. I pulled develop into this branch and now it's gone.